### PR TITLE
Update FortiAuthenticator integration to version 1.0.4

### PR DESCRIPTION
**Pull request by jasonlo82**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Bug fix

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ilappe** [comment]
@jasonlo82 
Thanks for your contribution!
it's look like your branch is not updated 
current pack version in master are 1.0.3 and in your branch it's 1.0.0
please merge from master and solve the conflicts

File: Packs/FortiAuthenticator/ReleaseNotes/1_0_4.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### FortiAuthenticator
+- Fixed an issue where the ***fortiauthenticator-update-user*** command for active status (true/false) update
```
Comment (ilappe) [inline Packs/FortiAuthenticator/ReleaseNotes/1_0_4.md:3]: Did you mean 
Fixed an issue where the ***fortiauthenticator-update-user*** command failed for active status (true/false) update

?

File: Packs/FortiAuthenticator/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -2,4 +2,7 @@
 ##### FortiAuthenticator
 - Fixed an issue where the ***fortiauthenticator-get-user*** command failed on LDAP user searches.
 - Added the arguments **username* and **token_serial** to the following commands: ***fortiauthenticator-update-user*** and ***fortiauthenticator-update-user***.
+<<<<<<< HEAD
+=======
 - Upgraded the Docker image to: *demisto/python3:3.9.6.24019*.
+>>>>>>> ca5de323bb1502d83a2f9b8e331d03d8f7bfb75d
```
Comment (ilappe) [inline Packs/FortiAuthenticator/ReleaseNotes/1_0_1.md:8]: Please drop this file from the PR
as the update release notes are the 1_0_4.md
and 1_0_1.md shouldn't be touched

**Review by ilappe — CHANGES_REQUESTED** [review]
Please see my comments 
in addition the build are red due to 
image

please run the command:
`demisto-sdk format -ud -i Packs/FortiAuthenticator/Integrations/FortiAuthenticator/FortiAuthenticator.yml` to fix this issue

File: Packs/FortiAuthenticator/ReleaseNotes/1_0_4.md (hunk 1 above)
Comment (jasonlo82) [reply Packs/FortiAuthenticator/ReleaseNotes/1_0_4.md:3]: Correct.

**Review by ilappe — APPROVED** [review]

**Pull request merged**